external/googletest-release-1.10.0/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright 2008, Google Inc.
```

Comment:
gitignore

Reply:
Did you attend Imed's lecture last Monday? 😁
You probably shouldn't touch license files or snippets. 😅

Reply:
@platisd 
I don't think it's been touched per se, it seems like everything has just been committed unnecessarily.

Reply:
The file you suggest removing states the following:

> Redistribution and use in source and binary forms, with or withoutmodification, are permitted provided that the following conditions aremet:
> * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.

It is literally illegal to redistribute `googletest` code without the particular file/copyright notice.

Reply:
@platisd 
honestly just landed my comment on there, since we we very quickly glanced over this PR over Zoom really quickly
Obviously leaving the license alone, would you recommend we still leave everything else, mock tests included, in the commit? Should none of the committed files be gitignored?

Reply:
@juliavankirk taking the circumstances into consideration, I'd say that commiting the `googletest` as they currently are in this pull request makes sense.

Reply:
@platisd by circumstances do you mean time constraints? I'd just like to know what best practices would be going forward, since I'm still learning as I go :)

Reply:
@platisd It would be great if you please give us a feedback and your opinion what to keep in mind in the future unit testing :)

Reply:
@juliavankirk the circumstances are the team's unfamiliarity with Cmake and therefore time.

In practice, even if the team was more experienced in Cmake (it's honestly not that hard btw but surely more complicated), the determining factor would be how easy it is for whatever build configuration system you are using to reliably fetch such external dependencies.
In professional projects I have seen such external dependencies fetched via Cmake (or equivalents), checked in like this pull request suggests and fetched as a git submodule.
In other words, all of these practices are potentially acceptable.

Reply:
@platisd I see, it's certainly the path of least resistance for us then. 
I'll keep your comments in mind going forward, thanks so much for explaining!

Reply:
@platisd 
hi again and sorry to be so annoying!  I promise to drop this line of questioning if you disapprove after this :)

Could we just delete all commits of googletest? Checking it all in is kind of messing with our stuff. Please see attached

<img width="592" alt="Screen Shot 2021-05-27 at 1 04 48 PM" src="https://user-images.githubusercontent.com/38304351/119816367-ea749400-beec-11eb-9d6b-8875509ddd75.png">

Proposed solution:
Provide instructions on how to get the code would suffice and be more clear for other developers and easier to maintain.  This way we can have someone download the repo from the source when they need it instead of keeping the tool in their repo. Maybe even integrate it into our Cmake build to do the download step automatically.

Reply:
Not sure what was crashing but as explained all three solutions are generally accepted.

Reply:
(My browser was crashing when I tried to make the review directly. Had to wait a few minutes to let it load everything before being able to click on the button after crashing, but waiting was the solution in this case :p)